goveralls.go

```diff
@@ -301,6 +302,10 @@ func process() error {
 		*service = "travis-ci"
 	}
 
+	if *service == "" && os.Getenv("GITHUB_EVENT_NAME") != "" {
+		*service = "GitHub Action"
```

Comment:
Should be `GitHub Actions` (with an ending `S`) right?

Reply:
Yes, right.